services/src/test-library/src/main/java/org/openkilda/testing/service/northbound/NorthboundService.java

```diff
@@ -76,7 +76,8 @@
 
     List<Long> installSwitchRules(SwitchId switchId, InstallRulesAction installAction);
 
-    List<Long> deleteSwitchRules(SwitchId switchId, DeleteRulesAction deleteAction);
+    List<Long> deleteSwitchRules(SwitchId switchId, DeleteRulesAction deleteAction, Integer inPort, Integer inVlan,
+                                 Integer outPort);
```

Comment:
Is there ability to delete by outVlan?

Reply:
It looks like not. Before creating this patch I checked the swagger documentation and found nothing about this.